src/status_im/contexts/wallet/tokens/events_test.cljs

```diff
@@ -0,0 +1,275 @@
+(ns status-im.contexts.wallet.tokens.events-test
+  (:require
+    [cljs.test :refer [deftest is testing]]
+    [status-im.contexts.wallet.tokens.events :as tokens.events]))
+
+(def eth
+  {:address  "0x0000000000000000000000000000000000000000"
+   :chainId  1
+   :decimals 18
+   :name     "Ether"
+   :symbol   "ETH"
+   :verified true})
+
+(def snt
+  {:address  "0x0000000000000000000000000000000000000001"
+   :chainId  1
+   :decimals 18
+   :name     "Status Network Token"
+   :symbol   "SNT"
+   :verified true})
+
+(def another-token1
+  {:address  "0x0000000000000000000000000000000000000002"
+   :chainId  1
+   :decimals 18
+   :name     "Another Token 1"
+   :symbol   "ANOTHER1"
+   :verified false})
+
+(def another-token2
+  {:address  "0x0000000000000000000000000000000000000000" ; uses the same address as `eth`
+   :chainId  1
+   :decimals 18
+   :name     "Another Token 2"
+   :symbol   "ANOTHER1" ;; uses the same symbol as `another-token1`
+   :verified false})
+
+
+(deftest store-token-list-test
```

Comment:
A minor detail @alwx. We have the macro `h/deftest-event`, which is to be preferred over defining vars just for event tests. You can check out the PR that introduced the macro for further details https://github.com/status-im/status-mobile/pull/20424